libsqlite3-sys/upgrade.sh

```diff
@@ -41,9 +41,10 @@ rm -rf "v${SQLCIPHER_VERSION}.tar.gz" sqlcipher.src
 
 # Regenerate bindgen file for sqlcipher
 rm -f "$SQLCIPHER_LIB_DIR/bindgen_bundled_version.rs"
+cargo clean
```

Comment:
This missing was a bug, and would cause some us to find the wrong bundled version some of the time.

Reply:
`find "$SCRIPT_DIR/../target" -type f -name bindgen.rs -exec rm {} \;` doesn't work ?

Reply:
I guess `cargo clean` at line 44 should be removed. And `# find "$SCRIPT_DIR/../target" -type f -name bindgen.rs -exec rm {} \;` at line 46 should be uncommented.

Reply:
oh, that makes sense.